content/en/docs/ops/common-problems/network-issues/index.md

```diff
@@ -81,7 +81,7 @@ Otherwise, the mode defaults to `DISABLE` causing client proxy sidecars to make
 instead of TLS encrypted requests. Thus, the requests conflict with the server proxy because the server proxy expects
 encrypted requests.
 
-To confirm there is a conflict, check whether the `STATUS` field in the output of the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-authn-tls-check) command
+To confirm there is a conflict, check whether the `STATUS` field in the output of the [`istioctl authn tls-check`](/docs/reference/commands/istioctl/#istioctl-experimental-authz-check) command
```

Comment:
This section of the doc needs to be reworked ... maybe there's a PR in the queue?

Reply:
Old commands shown on page will still work (they are deprecated). I am just updating the link to point to the new replacement command's ref.

Reply:
Where did you find this replacement? I'm pretty sure there is no replacement for `istioctl authn tld-check`. The following command in the text section won't work, as I understand it.